about.html

```diff
@@ -18,15 +18,13 @@ <h1>Robogyan</h1>
         <div class="navbar">
           <ul class="navbar-links">
             <li>
-              <a href="index.html">Home</a>
+              <a href="./index.html">Home</a>
```

Comment:
Generally asking, what does this do?

Reply:
this is best way to write this called it as relative path